string ctrlStop
string ctrlStart
---
bool stopSuccess
bool startSuccess